packages/prosess/prosess-vedtak/src/VedtakProsessIndex.stories.tsx

```diff
@@ -251,6 +251,53 @@ export const TeksterForAksjonspunkterSomSaksbehandlerMåTaStillingTil: Story = {
   },
 };
 
+export const OppgaverForAksjonspunkterSomSaksbehandlerMåTaStillingTil: Story = {
+  args: {
+    behandling: {
+      ...defaultBehandling,
+      aksjonspunkt: [
+        ...defaultAksjonspunkter,
+        {
+          definisjon: AksjonspunktKode.VURDERE_ANNEN_YTELSE,
+          status: AksjonspunktStatus.OPPRETTET,
+          begrunnelse: undefined,
+          kanLoses: false,
+          toTrinnsBehandling: true,
+        },
+        {
+          definisjon: AksjonspunktKode.VURDERE_DOKUMENT,
+          status: AksjonspunktStatus.OPPRETTET,
+          begrunnelse: undefined,
+          kanLoses: false,
+        },
+      ],
+    },
+    oppgaver: [
+      {
+        oppgavetype: 'VUR_KONS_YTE',
```

Comment:
Her kan du evt laga ein kodeverk-enum som du brukar i staden for hardkoda strengar. Men det blir ein vurdering. Om desse kodane til dømes kun blir brukt her, så er det ikkje så mykje vits i